### PR TITLE
Fix issue report formatting

### PR DESCRIPTION
**Pull request by vincentmacri**
Currently the bug report and failure to build report issue templates have the environment sections set to render as Markdown. Rather than actually rendering this as Markdown, GitHub renders these as Markdown source code which I imagine is not what was intended for these templates. This PR removes the instructions to render as Markdown source code, which has the effect of the section rendering as processed Markdown.

Examples of current formatting: https://github.com/vincentmacri/sage/issues/6 https://github.com/vincentmacri/sage/issues/7

Examples of new proposed formatting: https://github.com/vincentmacri/sage/issues/3 https://github.com/vincentmacri/sage/issues/5

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation and checked the documentation preview.

### :hourglass: Dependencies
None

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/failure_building_from_source.yml [hunk 1]
```diff
@@ -17,9 +17,8 @@ body:
           - **OS**: Ubuntu 20.04
           - Sage Version: 9.2
       value: |
-          - **OS**:
+          - **OS**: 
           - **Sage Version**:
```
Comment (kwankyu) [inline .github/ISSUE_TEMPLATE/failure_building_from_source.yml:21]: seems an extra space is added in L20.

**Comment by kwankyu** [comment]
Nice catch. I wonder why this was not reported earlier...

File: .github/ISSUE_TEMPLATE/failure_building_from_source.yml (hunk 1 above)
Comment (vincentmacri) [reply .github/ISSUE_TEMPLATE/failure_building_from_source.yml:21]: That was intentional, there should be a space between the colon and the OS name. This just saves the report submitter from having to add the space themselves.

We want

- **OS**: Ubuntu 20.04

not

- **OS**:Ubuntu 20.04

Comment (kwankyu) [reply .github/ISSUE_TEMPLATE/failure_building_from_source.yml:21]: Ah, right. I thought only that line has a space, inconsistently.

OK.

**Review by kwankyu — APPROVED** [review]
Thanks.

**Pull request merged**